.github/workflows/test.yml

```diff
@@ -1,12 +1,12 @@
-name: Build
+name: Test
```

Comment:
I guess we may revert name back, there is no reason to rename this job

Reply:
https://github.com/qdrant/qdrant-dotnet/pull/48/commits/65c68a95575fc851ef7dac95874573ce65c93f0a